### PR TITLE
Add custom changes to instructor emails

### PR DESCRIPTION
**Pull request by cocococosti**
This PR adds customs changes to the text included in the emails sent from the instructor tab.

The text included is as requested by UTEC.

Screenshot_2020-12-10 Has sido invitado a inscribirte a nueva plantilla de certificados utec marzo 2020 - costi abarca gmai
Screenshot_2020-12-10 Te han dado de baja en nueva plantilla de certificados utec marzo 2020 - constanza abarca edunext co
Screenshot_2020-12-10 Te has inscrito a nueva plantilla de certificados utec marzo 2020 - constanza abarca edunext co - edu

@felipemontoya @jfavellar90

### CODE REVIEW

**Review by morenol — APPROVED** [review]

File: edx-platform/utec/lms/templates/instructor/edx_ace/enrollenrolled/email/body.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                {% endblocktrans %}
+            </h1>
+            
+            <p style="color: rgba(0,0,0,.75);">
+                Hola {{ full_name }},
+                <br />
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                Has sido inscripto por invitación en el curso “{{ course_name }}”. El curso estará visible una vez que accedas, en el panel principal de la plataforma EDU con tu usuario.
```
Comment (jfavellar90) [inline edx-platform/utec/lms/templates/instructor/edx_ace/enrollenrolled/email/body.html:21]: typo: inscripto

File: edx-platform/utec/lms/templates/instructor/edx_ace/enrolledunenroll/email/body.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                {% endblocktrans %}
+            </h1>
+
+            <p style="color: rgba(0,0,0,.75);">
+                Hola {{ full_name }},
+                <br />
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                Has sido desinscripto del curso “{{ course_name }}” por un moderador. Este curso ya no aparecerá en tu panel principal.
```
Comment (jfavellar90) [inline edx-platform/utec/lms/templates/instructor/edx_ace/enrolledunenroll/email/body.html:21]: typo: desinscripto

File: edx-platform/utec/lms/templates/instructor/edx_ace/allowedunenroll/email/body.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                {% blocktrans %}You have been unenrolled from {{ course_name }}{% endblocktrans %}
+            </h1>
+
+            <p style="color: rgba(0,0,0,.75);">
+                Hola,
+                <br />
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                Has sido desinscripto del curso “{{ course_name }}” por un moderador. Este curso ya no aparecerá en tu panel principal.
```
Comment (jfavellar90) [inline edx-platform/utec/lms/templates/instructor/edx_ace/allowedunenroll/email/body.html:19]: typo: desinscripto

**Comment by jfavellar90** [comment]
@cocococosti le doy approve de una vez, el resto Ok

**Review by jfavellar90 — APPROVED** [review]

File: edx-platform/utec/lms/templates/instructor/edx_ace/enrollenrolled/email/body.html (hunk 1 above)
Comment (cocococosti) [reply edx-platform/utec/lms/templates/instructor/edx_ace/enrollenrolled/email/body.html:21]: not a typo, that is how they say it in some places @jfavellar90 it was like that in UTEC requirements

File: edx-platform/utec/lms/templates/instructor/edx_ace/enrolledunenroll/email/body.html (hunk 2 above)
Comment (cocococosti) [reply edx-platform/utec/lms/templates/instructor/edx_ace/enrolledunenroll/email/body.html:21]: not a typo, that is how they say it in some places @jfavellar90 it was like that in UTEC requirements

File: edx-platform/utec/lms/templates/instructor/edx_ace/allowedunenroll/email/body.html (hunk 3 above)
Comment (cocococosti) [reply edx-platform/utec/lms/templates/instructor/edx_ace/allowedunenroll/email/body.html:19]: not a typo, that is how they say it in some places @jfavellar90 it was like that in UTEC requirements

**Comment by jfavellar90** [comment]
La verdad no tenía idea que las dos formas eran correctas: https://www.rae.es/dpd/inscribir. Ahora soy un poco menos bruto

File: edx-platform/utec/lms/templates/instructor/edx_ace/allowedunenroll/email/body.html (hunk 3 above)
Comment (jfavellar90) [reply edx-platform/utec/lms/templates/instructor/edx_ace/allowedunenroll/email/body.html:19]: La verdad no tenía idea que las dos formas eran correctas: https://www.rae.es/dpd/inscribir. Ahora soy un poco menos bruto

**Pull request merged**